micropub.php

```diff
@@ -337,7 +337,8 @@ public static function generate_post_content($args) {
     if (current_theme_supports('microformats2')) {
       return $args;
 		}
-    // Disable if Kind Taxonomy is enabled
+    // Disable if Kind Taxonomy is enabled. The Kind Taxonomy is used in the Post Kinds plugin.
+    // The Post Kinds plugin handles its own markup. 
```

Comment:
the valuable part to add here is the link, eg https://wordpress.org/plugins/indieweb-post-kinds/